src/main/java/cc/hyperium/mods/playtime/PlayTime.java

```diff
@@ -1,25 +1,24 @@
 package cc.hyperium.mods.playtime;
 
 import cc.hyperium.Hyperium;
-import cc.hyperium.event.*;
-import cc.hyperium.internal.addons.IAddon;
+import cc.hyperium.config.Settings;
+import cc.hyperium.event.EventBus;
+import cc.hyperium.event.InvokeEvent;
+import cc.hyperium.event.TickEvent;
 import cc.hyperium.mods.AbstractMod;
 
-import java.sql.Time;
-import java.time.Duration;
-import java.time.LocalDateTime;
-import java.time.ZoneId;
-import java.time.ZonedDateTime;
-import java.util.Calendar;
-import java.util.Date;
-import java.util.Timer;
-import java.util.TimerTask;
-
-public class PlayTime extends AbstractMod  {
+public class PlayTime extends AbstractMod {
+    static long startSysTime;
+    static long startConfigTime;
+    static long sessionPlayTime;
+    static long totalPlayTime;
```

Comment:
Is there a reason these are static?

Reply:
all of these values are no longer static

--unresolve if you want to